src/plugins/condition/Condition.js

```diff
@@ -180,6 +181,7 @@ export default class ConditionClass extends EventEmitter {
         if (found) {
             this.criteria[found.index] = criterion.data;
             this.subscribe();
+            // TODO nothing is listening to this
```

Comment:
We can get rid of this if its not being used.

Reply:
@shefalijoshi , see above about TODOS.

Reply:
It's ok to go ahead and remove comments or change code. This is why we have code reviews.